From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Sat, 8 Feb 2020 16:06:57 +0100
Subject: ANDROID: dummy-cpufreq: fix up 5.6-rc1 API changes

The cpufreq api changed, so handle it properly in dummy-cpufreq so that
things keep building.

Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: Iaf496cf2777bf0f7fceb388c8d3baabad232b576
---
 drivers/cpufreq/dummy-cpufreq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/cpufreq/dummy-cpufreq.c b/drivers/cpufreq/dummy-cpufreq.c
--- a/drivers/cpufreq/dummy-cpufreq.c
+++ b/drivers/cpufreq/dummy-cpufreq.c
@@ -28,7 +28,7 @@ static unsigned int dummy_cpufreq_get(unsigned int cpu)
 	return 1;
 }
 
-static int dummy_cpufreq_verify(struct cpufreq_policy *policy)
+static int dummy_cpufreq_verify(struct cpufreq_policy_data *data)
 {
 	return 0;
 }
